DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionTask+Tracking.swift

```diff
@@ -0,0 +1,38 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+private var sessionFirstPartyHostsKey: UInt8 = 1
```

Comment:
I'm wondering if we should seek for some unique numbers for all those associated objects keys.

I imagine that some apps might use exactly the same key which might lead to unexpected behaviour

Reply:
there is a remote chance of colliding, so I changed the numbers to other than 1,2,3 etc.